app/scripts/components/facets/templates/facet.html

```diff
@@ -70,6 +71,16 @@
           {{ ::term.key | translate }}
         </span>
       </span>
+      <span data-ng-if="term.key === '_missing'"
```

Comment:
Yup, this is how we can show "No Data" instead of "MISSING". We could translate this in the controller but, since this is a presentation, I do it in here even it involves a bit boilerplate.